go/merkletree/merkle_verifier_test.go

```diff
@@ -1,10 +1,13 @@
 package merkletree
 
 import (
+	".."
```

Comment:
This line imports the `ct` lib. it could be rewritten as 
`import ct ..` or 
`import ct github.com/google/certificate-transparency/go`

What's your preference?